modules/gapi/src/streaming/onevpl/demux/async_mfp_demux_data_provider.cpp

```diff
@@ -36,7 +36,7 @@ static HRESULT create_media_source(const std::string& url, IMFMediaSource **ppSo
         hr = MFCreateSourceResolver(&source_resolver_tmp);
         if (FAILED(hr)) {
             throw DataProviderSystemErrorException(HRESULT_CODE(hr),
-                                                   "cannot create MFCreateSourceResolver from URI: " +
+                                                   "cannot create MFCreateSourceResolver from URL: " +
```

Comment:
it's not URL, but URI: Universal Resource Identificator. It means that not only a filename maybe here. I'm not sure about network path or stream ID but it may work out according to Microsoft docs

to exaggerate the last failed case (with `-input`) maybe it had better to change it to  
"cannot create MFCreateSourceResolver from URI: \"" + url + "\""